docs/sources/reference/config-blocks/livedebugging.md

```diff
@@ -0,0 +1,40 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/config-blocks/livedebugging/
+description: Learn about the livedebugging configuration block
+menuTitle: livedebugging
+title: livedebugging block
+---
+
+<span class="badge docs-labels__stage docs-labels__item">Experimental</span>
+
+# livedebugging block
+
+{{< docs/shared lookup="stability/experimental.md" source="alloy" version="<ALLOY_VERSION>" >}}
+
+`livedebugging` is an optional configuration block that enables the [live debugging feature][debug], which streams real-time data from your components directly to {{< param "PRODUCT_NAME" >}}'s UI.
+
+By default, [live debugging][debug] is disabled and must be explicitly enabled through this configuration block to make the debugging data visible in the UI.
```

Comment:
```suggestion
By default, [live debugging][debug] is disabled and must be explicitly enabled through this configuration block to make the debugging data visible in the Alloy UI.
```

Reply:
This one should be {{< param "PRODUCT_NAME" >}}, no?

Reply:
Yes.... I clicked through too quickly